### DISCUSSION TITLE
HTMX FAQ

### DISCUSSION THREAD

**Discussion by 1cg** [bigskysoftware/htmx#679] [Q&A] [answered]
# General

### "How is htmx spelled?"

htmx, all lower case, is the correct spelling for htmx.

### "Is there a chat for htmx?"

Yes, the [htmx discord](https://htmx.org/discord) is quite active.

### "What is rule 7b?"

Rule 7b, when invoked, means that you should try to implement your suggestion.  The htmx codebase is pretty small and manageable, and you will be surprised how your idea could potentially be.  Extensions are also a good place to look at for implementing 7b's.  The
rule is a running joke on the htmx discord: "Hey, htmx should do X"  "Sounds great.  I invoke rule 7b."

# htmx

### "How do I update content on the page that isn't in the target?"

See [Updating Other Content](https://htmx.org/examples/update-other-content/).

# _hyperscript

### "What is the relationship between the htmx and _hyperscript projects?  Do I have to use _hyperscript?"

htmx and [_hyperscript](https://hyperscript.org) are sibling projects.  They are designed to work well together, htmx for server-interactions and _hyperscript for pure front-end interactions, but **neither requires the other**.  You may use Vanilla JS, jQuery, Alpine.js or any other library you'd like for your front end scripting needs.

**Comment by 1cg** [comment, marked as answer]
:point_up:

**Reply by alexander-bruun** [reply]
@1cg Is the discord still alive? The invite link seems to be dead.
